draft-rbw-home-servers.md

```diff
@@ -206,25 +218,27 @@ R-REVOKE-AUTH:
 
 ## Delegated Credentials {#delegated}
 
-Delegated credentials {{?RFC9345}} allows the entity operating the CPE
-(e.g., vendor or ISP) to sign a 7-day validity for the CPE's public key.
-The frequency of CA interactions remains the same as with normal
-certificates ({{normal-certificates}}), but the interactions are with
-the vendor's service rather than the public CA.
-
-As currently specified in {{?RFC9345}}, the same name would be issued to all CPEs
-making it impossible to identify whether the delegated credential is
-issued to the intended CPE or an "evil-twin" CPE. This drawback can be corrected
-by enhancing {{?RFC9345}} to include a string that uniquely identifies the
-delegated credential (e.g., including hash of customer id or other unique identifier
-in the FQDN such as "HASH.cpe.example.com").
+Delegated credentials {{?RFC9345}} allows the entity operating the
+device (e.g., vendor or ISP) to sign a 7-day validity for the device's
+public key (Short-Term Automatically Renewed (STAR)).  The frequency of CA interactions remains the same as with
+normal certificates ({{normal-certificates}}).  For each device, the
+interactions are with the vendor's service rather than with the public
+CA.
```

Comment:
In case of RFC 9345, there is no additional interaction with the CA. You seem to referring to STAR defined in ACME.

Reply:
RFC9345 recommends 7-day, but I guess you are expecting a system with a longer lifetime -- if so, we need to say so in this I-D.  How many days would you like?

Reply:
I came up with even softer language.  Now reads:

> Delegated credentials {{?RFC9345}} allows the entity operating the CPE (e.g., vendor or ISP) to sign the CPE's public key. The frequency of CA interactions for renewals can remain the same as with normal certificates ({{normal-certificates}}), but the interactions are with the vendor's service rather than the public CA.